plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemHornInstrument.java

```diff
@@ -0,0 +1,80 @@
+package com.denizenscript.denizen.objects.properties.item;
+
+import com.denizenscript.denizen.objects.ItemTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.Material;
+import org.bukkit.MusicInstrument;
+import org.bukkit.inventory.meta.MusicInstrumentMeta;
+
+public class ItemHornInstrument extends ItemProperty {
+
+    public static boolean describes(ItemTag item) {
+        return item.getBukkitMaterial() == Material.GOAT_HORN;
+    }
+
+    @Override
+    public String getPropertyString() {
+        MusicInstrument instrument = getMusicInstrument();
+        if (instrument != null) {
+            return Utilities.namespacedKeyToString(instrument.getKey());
+        }
+        return null;
+    }
+
+    @Override
+    public String getPropertyId() {
+        return "horn_instrument";

```

Comment:
Might be a good idea to make it just `instrument` (and `ItemInstrument` respectively), so that if a different type of item ever has an instrument or something of that sort it can just be added into this property